cloudbuild.yaml

```diff
@@ -5,17 +5,17 @@
 steps:
 
   # Build app_flutter.
-  - name: gcr.io/flutter-dashboard/flutter
+  - name: gcr.io/$PROJECT_ID/flutter
     entrypoint: '/bin/bash'
     args: ['cloud_build/app_flutter_build.sh']
 
   # Build angular dart app.
-  - name: gcr.io/flutter-dashboard/flutter
+  - name: gcr.io/$PROJECT_ID/flutter
     entrypoint: '/bin/bash'
     args: ['cloud_build/app_build.sh']
 
   # Deploy a new version to google cloud.
   - name: 'gcr.io/cloud-builders/gcloud'
     entrypoint: '/bin/bash'
-    args: ['cloud_build/deploy.sh', 'flutter-dashboard', '$SHORT_SHA']
+    args: ['cloud_build/deploy.sh', '$PROJECT_ID', '$SHORT_SHA']
```

Comment:
How are we providing PROJECT_ID?

Reply:
It's one of the default substitutions provided by cloud build: https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values